boutdata/squashoutput.py

```diff
@@ -76,7 +82,7 @@ def squashoutput(datadir=".", outputname="BOUT.dmp.nc", format="NETCDF4", tind=N
         datadirnew = tempfile.mkdtemp(dir=datadir)
         for f in glob.glob(datadir + "/BOUT.dmp.*.??"):
             if not quiet:
-                print("moving", f)
+                print("moving", f, flush=True)
```

Comment:
Would a better design be to use `logging` in the future?

Reply:
I don't know the `logging` module, so I don't know either way. The motivation for the change is for when `squashoutput()` is being run in a batch job - without `flush=True` the output is only flushed to the log at the end of the script, so it's not useful for monitoring the progress of the job.

Reply:
Makes sense. On can achieve the same with [`logging`](https://docs.python.org/3/library/logging.html), which also has the advantage that the output does not to be ephemeral